### DISCUSSION TITLE
Different style and options types on installments from different cards

### DISCUSSION THREAD

**Discussion by joxpulp** [mercadopago/sdk-react#39] [Ideas]
Hi, here again LOL, I was trying different credit cards from the [test payment cards](https://www.mercadopago.com.ar/developers/es/docs/checkout-bricks/integration-test/test-cards), and notice that the option to choose the installments are different

When using a Visa and MasterCard credit cards, is a select:

<img width="629" alt="Captura de Pantalla 2023-04-25 a la(s) 06 17 38 p  m" src="https://user-images.githubusercontent.com/73070103/234412736-773e90bd-a2c4-4048-b5fb-7d52a217b98b.png">

When using an American Express credit card, changes to radio buttons:

<img width="651" alt="Captura de Pantalla 2023-04-25 a la(s) 06 17 18 p  m" src="https://user-images.githubusercontent.com/73070103/234412784-f49e8bee-eb5a-4ea5-8394-fc998fd0ec6b.png">

Maybe this could be confusing to the user, since it brings more information to the form when using American Express, in the other hand I think that is easy to read and choose using the select from Visa and MasterCard.  

Is there a way to make the select as a default type of 'style' to select an installment for all the cards and avoid the style changing for American Express?

**Comment by meliguicarvalho** [comment]
Hello! Thanks for the suggestion!

This difference is due to the amount of installments.
The radio button shows more complete information and has a more distinct appearance, but it occupies more space, so if there are less than 6 installment options available, we show the radio button, otherwise it is the select.
Hope this clarifies the behavior.

Your suggestion is interesting, we will analise the possibility of allowing the selection of a default mode, and give you feedback in this thread.

**Comment by AleFossati** [comment]
Hello @joxpulp, we have an update about this matter.
We have discussed with UX team and the analysis conclusion is that it's important to keep this behavior (radiobutton for less than 6 installments, dropdown for 6 or more). As a result, we have no intention to create a customization allowing to choose between one of these elements.
Is this a mandatory feature for your project?

**Reply by joxpulp** [reply]
Thanks, it's not mandatory in our project, but at first glance it was a bit weird for me that the way the installments are displayed has such a different change depending on the number of installments.

**Reply by AleFossati** [reply]
Thanks for your feedback, it help us a lot.
For now, I'll tag this discussion as resolved.

**Comment by wemerson-silva-kz** [comment]
how you change styles in fields??

**Reply by AleFossati** [reply]
fields inherit styles from brick's parent html element
